config/job.js

```diff
@@ -65,12 +66,13 @@ const SCHEMA_SETTINGS = Joi.object().optional();
 const SCHEMA_TEMPLATE = Joi.string().regex(Regex.FULL_TEMPLATE_NAME);
 const SCHEMA_JOB = Joi.object()
     .keys({
-        steps: SCHEMA_STEPS,
+        annotations,
```

Comment:
@r3rastogi I think what he meant is this:
```
> var annotations = {"annotations": true}
undefined
> var obj = {annotations}
undefined
> obj
{ annotations: { annotations: true } }
> 
```
But we actually do not want the extra layer.